src/NJsonSchema/Infrastructure/XmlDocumentationExtensions.cs

```diff
@@ -562,6 +562,11 @@ private static async Task<string> GetXmlDocumentationPathAsync(dynamic assembly)
             }
             catch
             {
+                var currentDirectory = await DynamicApis.DirectoryGetCurrentDirectoryAsync();
```

Comment:
Please move this code into the try{} block. This method must never throw an exception...

Reply:
Added a new try/catch to get current directory path. In case any error happens, it will return null.

Reply:
Added a new try/catch to get current directory path. In case any error happens, it will return null.